packages/langium-cli/src/generator/ast-generator.ts

```diff
@@ -74,7 +76,7 @@ function generateAstReflection(config: LangiumConfig, astTypes: AstTypes): Gener
 
     reflectionNode.append(
         '}', NL, NL,
-        `export const reflection = new ${config.projectName}AstReflection();`, NL
+        `export const reflection = new ${config.projectName}AstReflection();`, NL, NL
```

Comment:
Why the additional newline? This leaves us with two blank lines at the end of files:

![image](https://github.com/langium/langium/assets/4377073/b636339c-4d7e-4e54-8923-6e127a2f6fc3)

Reply:
Weird about the two empty lines, I might used the wrong line to add an NL....